cluster/aws/templates/format-disks.sh

```diff
@@ -172,7 +172,10 @@ if [[ ${docker_storage} == "btrfs" ]]; then
   DOCKER_OPTS="${DOCKER_OPTS} -s btrfs"
 elif [[ ${docker_storage} == "aufs-nolvm" || ${docker_storage} == "aufs" ]]; then
   # Install aufs kernel module
-  apt-get install --yes linux-image-extra-$(uname -r)
+  #apt-get install --yes linux-image-extra-$(uname -r)
```

Comment:
This line should not be commented out!

You need aufs module for the current kernel, so that kubernetes could run docker images until kernel is upgraded and machine is reboot.

I made the same mistake when trying to fix the root cause but I had to leave two separate `apt-get install linux-image-extra-XXX`.